inputs.py

```diff
@@ -0,0 +1,32 @@
+from brownie import convert
+from flask import request
+
+
+def method_input():
+    choices = ['FIFO','LIFO']
+    assert request.json['type'] in choices
+    return request.json['type']
+
+
+def address_inputs():
+    good_list, bad_list = [], []
+    for address in request.json['addresses']:
+        clean = clean_address(address)
+        if clean: good_list.append(clean)
+        else: bad_list.append(address)
+    return (good_list, bad_list)
+
+
+def clean_address(address: str):
+    if is_address(address): return convert.to_address(address)
+    
+    
+def is_address(address: str) -> bool:
+    return all([
+        len(address) == 42
```

Comment:
I'd go for the validations from brownie.convert which are more robust.
Consider this:
`convert.to_address('0x' + '.' * 40) // len(address) == 42`

you could remove #is_address altogether and move up to the #clean_address with:
```
try:
    return convert.to_address(address)
except ValueError
    return None
```

then you can can check for:
```
if clean_address(address): ....
else: ...
```

Reply:
we also use the brownie convert function as a check elsewhere in the code, just not in this part
if brownie is unable to convert to checksum, the address will be added to bad_addresses

Reply:
the only thing, I'd always prefer the target system's validation over self-built ones.
So usually if you use brownie's convert address to determine if the address is valid, the validations will be more robust than re-inventing the whell...